src/elexmodel/models/BootstrapElectionModel.py

```diff
@@ -888,6 +888,9 @@ def compute_bootstrap_errors(
             n_feat_ignore_reg=1,
         )
 
+        print("ols coefficients, normalized margin: ", ols_y.coefficients.flatten())
+        print("bootstraped ols coefficients, normalized margin: ", ols_y_B.coefficients.mean(axis=-1))
+
```

Comment:
Are these two statements something we'd want to log in production?  Or are they intended for debugging the model offline? 🤔

Reply:
Nevermind, they became log statements in the next commit!  Sorry 😅 🎉